src/api/win32/mod.rs

```
@@ -218,7 +229,10 @@ impl Window {
     }
 
     pub fn set_cursor(&self, _cursor: MouseCursor) {
-        unimplemented!()
+        // XXX Ignore cursor changes for now on Windows, but don't crash; this
+        // should be implemented using SetCursor and LoadCursor after mapping _cursor
+        // to Windows cursors.
+        //unimplemented!()
```

I don't like this change. It just fails silently and then users spend time trying to understand why things don't work.